### PR TITLE
Adds an endpoint to post cluster peers to

### PR DESCRIPTION
**Pull request by replay**

### CODE REVIEW

File: api/cluster.go [hunk 1]
```diff
@@ -51,6 +51,41 @@ func (s *Server) getClusterStatus(ctx *middleware.Context) {
 	response.Write(ctx, response.NewJson(200, status, ""))
 }
 
+func (s *Server) postClusterMembers(ctx *middleware.Context, req models.ClusterMembers) {
+	memberNames := make(map[string]struct{})
+	toJoin := make([]string, 0)
```
Comment (Dieterbe) [inline api/cluster.go]: you don't need to pre-allocate slices. just declaring them is enough. len, range, etc will work fine

Comment (replay) [inline api/cluster.go]: thx, fixing

**Comment by Dieterbe** [comment]
LGTM now. @replay shall i merge?

**Comment by replay** [comment]
yes please @Dieterbe

**Pull request merged**